src/main/java/core/basesyntax/BallNumberSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class BallNumberSupplier {
+    public int getRandomNumber() {
+        return new Random().nextInt(100);

```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants